### ISSUE TITLE
Effects on react-native Animated

### ISSUE THREAD

**Issue by RWOverdijk** [software-mansion/react-native-reanimated#231]
I'm using react-navigation and react-native's Animated currently. Now I started using https://github.com/react-navigation/react-navigation-animated-switch I'm also using react-native-animated. (added an issue on https://github.com/react-navigation/react-navigation-animated-switch/issues/1 as well)

I'm not 100% sure where to post this issue but the effects feel like this is the place to be.

```ts
export const MainNavigator = createAnimatedSwitchNavigator({
  Onboarding: OnboardingScreen,
  App: AppScreen,
}, {
  transition: (
    <Transition.Together>
      <Transition.Out
        type="slide-left"
        durationMs={10000}
        interpolation="easeIn"
      />
    </Transition.Together>
  ),
});
```

My OnboardingScreen has a bunch of animations _(reacct-native's animated animations)_ running in a loop. As soon as I navigate from one screen to the next, the transition as configured above kicks in (as it should).

The actual issue I'm seeing is that it messes up my react-native Animated animations. They seem to jump forward in time and then freeze. The slide works perfectly, the rest is just jumpy.

**TL;DR;** Before I dive into this too deep... Are there any known conflicts between react-native-reanimated and react-native's Animated that could cause strange side-effects?

**Comment by jakub-gonet** [comment]
Hi @RWOverdijk,
sorry for the late response

Is this issue still relevant or can we close it?
